### DISCUSSION TITLE
您好，我已经购买专业版，苹果手机IOS无法登录专业版。请帮忙给我的苹果手机授权一下。我的账户是：yuyalun100@163.com。谢谢

### DISCUSSION THREAD

**Discussion by aaron100100100** [getlantern/lantern-forum-cn#233] [Q&A]
您好，我已经购买专业版，电脑可以正常登录使用，但是，苹果手机IOS无法登录专业版。请帮忙给我的苹果手机授权一下。我的手机已经通过国外的商店下载了蓝灯APP。

我的账户是：yuyalun100@163.com。谢谢

我是今天2024.03.11日购买的。

**Comment by MaxMKMax** [comment]
@aaron100100100 您好, 谢谢您的反馈. 好的明白了! 我帮您和团队说明这个问题。我帮你紧急解决问题。让我通知技术团队! 他们会手动帮助您升级到专业版。非常抱歉影响了您的体验感, 谢谢您对蓝灯的支持。

**Reply by MaxMKMax** [reply]
@aaron100100100 晚上好! 我已经通知负责人处理您的请求。由于其他用户有很多请求票，请给我们 1 或 2 天的时间。我们将尽快处理您的问题。再次感谢您对蓝灯笼的支持。我们很高兴拥有像您这样的宝贵用户。谢谢!

您也可以尝试通过下面的常见问题解答链接自行解决问题. 看看是否有帮助. 
https://lantern.io/zh-Hans/faq

**Comment by aaron100100100** [comment]
你好，我的苹果手机的“蓝灯”软件，现在还是不能用。你上次回复我的邮件说3.14号能授权成功，现在都3.17号了，手机还是不能用！

既然你要授权给我的手机，世界上那么多苹果手机，你怎么知道授权哪台手机呢？这个怎么去授权？麻烦尽快给处理一下！谢谢

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年03月12日 00:00 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 晚上好! 我已经通知负责人处理您的请求。由于其他用户有很多请求票，请给我们 1 或 2 天的时间。我们将尽快处理您的问题。再次感谢您对蓝灯笼的支持。我们很高兴拥有像您这样的宝贵用户。谢谢!

您也可以尝试通过下面的常见问题解答链接自行解决问题. 看看是否有帮助.
https://lantern.io/zh-Hans/faq

**Reply by MaxMKMax** [reply]
@aaron100100100  您好。当前版本仍然存在登录问题。请给我们几天的时间来解决，我们会尽快更新您的信息。我们将很快发布最新版本。抱歉让您久等了。稍后我们还将补偿您更长的专业版。

**Comment by aaron100100100** [comment]
好的，谢谢！请尽快解决苹果手机登录和授权的问题。解决后，请给发个邮件告知一下。谢谢

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年03月17日 23:48 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 您好。当前版本仍然存在登录问题。请给我们几天的时间来解决，我们会尽快更新您的信息。我们将很快发布最新版本。抱歉让您久等了。稍后我们还将补偿您更长的专业版。

**Reply by MaxMKMax** [reply]
@aaron100100100 好的. 我们今晚会讨论解决方案. 尽快给您答复.

**Comment by cyq455** [comment]
你好，我的苹果手机的“蓝灯”软件，现在还是不能用。麻烦尽快给处理一下！谢谢

**Reply by MaxMKMax** [reply]
@cyq455 嗯嗯好的. 另一个帖子回复您了. 非常抱歉影响了您的体验感. 我们会尽快修复!

**Comment by aaron100100100** [comment]
你好，我的苹果手机仍旧无法安装“蓝灯”app，一直闪退！请问，什么时候能苹果手机可以登录啊？


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年03月17日 23:48 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 您好。当前版本仍然存在登录问题。请给我们几天的时间来解决，我们会尽快更新您的信息。我们将很快发布最新版本。抱歉让您久等了。稍后我们还将补偿您更长的专业版。

**Comment by MaxMKMax** [comment]
@aaron100100100 这个新版本应该修复了闪退和别的问题了. 请您下载新版本. 基本上是没看到有iOS用户有闪退的问题了. 谢谢您.

https://github.com/getlantern/lantern

**Comment by aaron100100100** [comment]
客服你好，之前反应的苹果手机版本问题一直没有解决。苹果手机版的蓝灯，还是5个月前的版本（参考图2）。


图3和4，是手机版本的页面截图，根本就没有任何可以操作的功能


图5和6，是电脑专业版的页面。特别是图6，我想给我的手机授权，需要一个6位数的授权码。这个授权码从苹果手机哪里能找到？苹果手机根本没有可以找到的入口。
图5里面，我授权了2台电脑，从电脑里是可以找到授权码，可以绑定的。




请尽快给个回复！！！
​

---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年04月14日 00:21 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 这个新版本应该修复了闪退和别的问题了. 请您下载新版本. 基本上是没看到有iOS用户有闪退的问题了. 谢谢您.

https://github.com/getlantern/lantern

**Reply by MaxMKMax** [reply]
@aaron100100100 您好对的. 完全理解. 技术团队还在弄IOS版本. 现有的手机IOS版本根本就没有授权码 , 暂时用不了了.
真的非常抱歉影响了您的体验感. 
请您等待一下. 我们尽快了. 谢谢!

**Comment by aaron100100100** [comment]
好的，很期待苹果IOS的蓝灯app，希望尽快弄好。如果还得等很久的话，后面能否再延长一下会员时间啊？这个希望你们为顾客考虑一下哦。


希望你们一切顺利，辛苦了。


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年04月19日 11:42 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 您好对的. 完全理解. 技术团队还在弄IOS版本. 现有的手机IOS版本根本就没有授权码 , 暂时用不了了.
真的非常抱歉影响了您的体验感.
请您等待一下. 我们尽快了. 谢谢!

**Reply by MaxMKMax** [reply]
@aaron100100100 我们也有打算要补偿您们IOS专业版客户. 之后我们会帮您重新延长专业版的时效. 非常感恩您能理解我们. 超级感谢您!!!

**Comment by JaniForAndroid** [comment]
我的ios手机没有登录入口是怎么回事啊？版本为7.5.1， 构建版本为14，sdk 7.6.14 @Mike777Khoo

**Comment by RRRRRui** [comment]
您好 @Mike777Khoo ，我的账户已升级为专业版，但在iOS上未找到任何登录或填写授权码的地方，请帮忙解决。

**Comment by aaron100100100** [comment]
我的也是这个情况，在手机ios系统里没有找到授权码！！！

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年06月23日 19:25 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

您好 @Mike777Khoo ，我的账户已升级为专业版，但在iOS上未找到任何登录或填写授权码的地方，请帮忙解决。

**Reply by MaxMKMax** [reply]
@aaron100100100 @RRRRRui @JaniForAndroid 
你们好. 请你们等等. 新版iOS这两周会放出来了. 现在暂时无法输入授权码的. 会尽快放出新版. 新版已弄好, 在测试了. 好了, 就会放出来给你们更新. 授权码问题就解决了.
不好意思影响了你们的体验感. 谢谢您的耐心等待也谢谢你们支持蓝灯!

**Reply by RRRRRui** [reply]
请问新版iOS什么时候能放出来，又过去了一个多月……

**Comment by xRA1N** [comment]
ios 等回复

**Comment by aaron100100100** [comment]
你好，我之前购买了蓝灯专业版，最近2个月开始电脑一直登录不上去，有时候登录上去后，页面打开的很慢！还提示流量已经用尽，请购买专业版！！！什么情况啊？请给查一下，给个回复！


我的手机版蓝灯，最近1个多月，也是无法正常登录，是什么原因，请给解释一下！谢谢


***@***.***，专业版。

**Reply by MaxMKMax** [reply]
@aaron100100100 您好. 请您提供我账号邮箱和支付单据. 估计有bug所以没了专业版.
您提供我资料之后我会帮您提交问题. 非常抱歉啊. 谢谢您支持蓝灯!

**Comment by aaron100100100** [comment]
***@***.***
支付凭据见附件。


那我专业版，估计有2个月一直没法使用，是不是使用时间，也要给我相应的延长2个月啊？
还有，我的苹果手机app是前一阵才登录的（你们后来才研发出来的app，之前是一直没法用的。），并且也有一个月是没办法登录的，这样的话，给延长多久呢？


希望这个使用截止时间，也给解决一下，多延长一下哈。谢谢！




---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年10月27日 22:28 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | aaron100100100 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] ***@***.***。谢谢 (Discussion #233) |

@aaron100100100 您好. 请您提供我账号邮箱和支付单据. 估计有bug所以没了专业版.
